packages/system/src/Item/Components/StockItemSelectModal/StockItemSelectModal.tsx

```diff
@@ -113,9 +136,11 @@ export const StockItemSelectModal = ({
           disabled={saving}
           variant="ok"
           onClick={async () => {
-            setSaving(true);
-            await onChange(itemIds);
-            setSaving(false);
+            if (selectedItems instanceof Array) {
```

Comment:
Why this check? When can `selectedItems` ever NOT be an Array?

Reply:
I'll put that one down to a previous version. It's deffo an array now.